Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -160,12 +160,12 @@ void JitArm64::bcx(UGeckoInstruction inst)
   if (js.op->branchIsIdleLoop)
   {
     // make idle loops go faster
-    ARM64Reg WA = gpr.GetReg();
+    ARM64Reg WA2 = gpr.GetReg();
```

Comment:
i assume this change is to quiet a shadowed variable warning. are you sure the behavior is correct? the following line uses `WA`.